crowdsourcehinter/static/js/src/crowdsourcehinter.js

```diff
@@ -0,0 +1,305 @@
+function CrowdsourceHinter(runtime, element, data){
+
+    var executeHinter = true;
+    var isShowingHintFeedback = false;
+    var voted = false;
```

Comment:
Helper functions are okay too (and good). E.g. voted() which checks for the class over many divs or similar.